zq2/docs/staking/index.md

```diff
@@ -0,0 +1,19 @@
+---
+id: staking/index
+title: Staking
+---
+
+# Staking Mechanism
+
+The staking mechanism supports two variants:
+
+## Staking Variants
+
+### 1. **Liquid Staking**
+
+- Users receive a **non-rebasing Liquid Staking Token (LST)** upon delegation.
+- The LST can be sent to the validator's contract to withdraw the staked amount along with the corresponding share of validator rewards.
```

Comment:
I think this isn't very clear. What about

```
- The tokens represent the delegator's share of the validator's stake. As the validator earns rewards, the value of the staking token will increase. The tokens are burned when the delegator withdraws their stake, in return for ZIL.
```

Reply:
Done